CONTRIBUTING.md

```diff
@@ -116,13 +115,14 @@ In order to help reviewers:
 - Provide [osc](https://github.com/openSUSE/osc) commands to test your changes if that is the case.
 - For visual or design-related pull requests:
   - add screenshots or GIFs,
-  - enable review app *
+  - enable review app*
 - For performance-related pull requests:
-  - add sample data in the review app and put links in the pull request's description *
+  - add sample data in the review app* and put links in the pull request's description
```

Comment:
Regarding this, I wouldn't limit this to performance-related pull requests. When reviewing visual changes, it's always nice to have some sample data, so the person submitted the pull request should also do it in this case.